OrbitGl/CaptureSerializer.cpp

```diff
@@ -138,12 +159,24 @@ ErrorMessageOr<void> CaptureSerializer::Load(const std::string& filename) {
     return ErrorMessage("Error opening the file for reading");
   }
 
-  try {
-    return Load(file);
-  } catch (std::exception& e) {
-    ERROR("Loading capture from \"%s\": %s", filename, e.what());
-    return ErrorMessage("Error parsing the capture");
+  return Load(file);
+}
+
+bool ReadMessage(google::protobuf::Message* message,
+                 google::protobuf::io::CodedInputStream* input) {
+  uint32_t message_size;
+  if (!input->ReadLittleEndian32(&message_size)) {
+    return false;
   }
+
+  char* buffer = new char[message_size];
```

Comment:
Is this a case for `make_unique_for_overwrite` that is currently in `OrbitLinuxTracing`? Maybe we should move it to `OrbitBase`?

Reply:
yes - it also provides scoping of the buffer

Reply:
It's done, thanks!